bitnami/elasticsearch/values.yaml

```diff
@@ -310,7 +307,7 @@ metrics:
     registry: docker.io
     repository: bitnami/elasticsearch-exporter
     tag: 1.0.2
-    pullPolicy: Always
+    pullPolicy: IfNotPresent
```

Comment:
Why did you change the `pullPolicy` in the **values.yaml**?

Reply:
Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images